gateway-messages/src/sp_to_mgs.rs

```diff
@@ -87,7 +87,12 @@ pub enum SpResponse {
     },
     SerialConsoleDetachAck,
     PowerState(PowerState),
-    SetPowerStateAck,
+    /// Indicates that a `SetPowerState` request was performed successfully
+    /// and resulted in a power state transition.
+    ///
+    /// If the SP is already in the desired power state, the
+    /// [`Self::PowerStateUnchanged`] response is returned instead.
```

Comment:
I don't feel strongly about it, but maybe we should mention that this was formerly known as `SetPowerStateAck`?

Reply:
Good call, I'll add a note!